src/libs/actions/Policy.js

```diff
@@ -1000,6 +1001,38 @@ function openWorkspaceInvitePage(policyID, clientMemberEmails) {
     });
 }
 
+/**
+ *
+ * @param {String} reportID
+ */
+function leaveRoom(reportID) {
+    API.write('LeaveRoom', {
+        reportID,
+    }, {
+        optimisticData: [
+            {
+                onyxMethod: CONST.ONYX.METHOD.SET,
+                key: `${ONYXKEYS.COLLECTION.REPORT}${reportID}`,
+                value: {
+                    stateNum: CONST.REPORT.STATE_NUM.SUBMITTED,
```

Comment:
@marcochavezf is there any reason that we use 

```tsx

                value: {
                    stateNum: CONST.REPORT.STATE_NUM.SUBMITTED,
                    statusNum: CONST.REPORT.STATUS.CLOSED,
                },

```

Than directly set the report to null?

cc: @parasharrajat

Reply:
Hi @wildan-m, iirc we just close the report (where this values represent a closed report) to hide instead of removing completely from Onyx. And I think that's because in case we receive an update for that report, we don't show it again

Also I noticed this code doesn't exist in main, and was changed in this PR https://github.com/Expensify/App/pull/27966/files#diff-8afe5b71ee0436c21364148c86dadd640f2bff3e3d916addbb1f1f6f7e5b6a43R1939-R1940